src/test/java/guitests/MenuControlTest.java

```diff
@@ -21,6 +22,8 @@ public void menuControlTest() {
         modelUpdatedEventTriggered = false;
         UI.events.registerEvent((ModelUpdatedEventHandler) e -> modelUpdatedEventTriggered = true);
         PanelControl panelControl = (PanelControl) find("#dummy/dummy_col0").getParent();
+        Preferences testPref = panelControl.getPreferences();
```

Comment:
You can exclude it [here](https://github.com/HubTurbo/HubTurbo/blob/master/config/findbugs/excludeFilter.xml#L23-L30).